assets/js/modules/analytics/datastore/settings.js

```diff
@@ -290,16 +290,16 @@ export const isGA4DashboardView = createRegistrySelector( ( select ) => () => {
 	const ga4ModuleActive =
 		select( CORE_MODULES ).isModuleActive( 'analytics-4' );
 
-	if ( ! ga4ModuleActive ) {
-		return false;
-	}
-
 	const dashboardView = select( MODULES_ANALYTICS ).getDashboardView();
 
-	if ( dashboardView === undefined ) {
+	if ( ga4ModuleActive === undefined || dashboardView === undefined ) {
```

Comment:
This change is required. Otherwise, it will return false when the modules still need to be resolved. That will cause the `UA` component to render initially and cause network failure if the `UA` module isn't connected.